judge/models/runtime.py

```diff
@@ -94,6 +95,10 @@ def get_python2(cls):
     def get_absolute_url(self):
         return reverse('runtime_list') + '#' + self.key
 
+    @classmethod
+    def get_default_language(cls):
+        return Language.objects.get(key=settings.DEFAULT_USER_LANGUAGE).pk
```

Comment:
This function implies that it returns an object. Either rename or make use lambda or do something else.

Reply:
After Django 1.7, you cannot use a lambda for `default`, so I guess I'll rename it to `get_default_language_pk`